ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-HWF-R2-CUI-nonprod.json

```diff
@@ -13,19 +13,7 @@
   },
   {
     "CaseTypeID": "CIVIL",
-    "CaseEventID": "PARTIAL_REMISSION_HWF_GRANTED",
-    "AccessControl": [
-      {
-        "UserRoles": [
-          "cui-admin-profile"
-        ],
-        "CRUD": "CRU"
-      }
-    ]
-  },
-   {
-    "CaseTypeID": "CIVIL",
-    "CaseEventID": "MORE_INFORMATION_HWF",
```

Comment:
even MORE_INFORMATION_HWF is not  needed , check is it replaced while resolving the conflicts